robottelo/config/__init__.py

```diff
@@ -15,7 +15,7 @@
     os.environ['ROBOTTELO_DIR'] = str(robottelo_root_dir)
 
 
-settings = LazySettings(
+settings = Dynaconf(
```

Comment:
I think we should continue to explicitly use LazySettings

Reply:
Its mentioned here: https://github.com/rochacbruno/dynaconf/blob/master/dynaconf/__init__.py#L19

Reply:
I don't see there that `LazySettings` is or will be deprecated, just that its the default alias.

That's why we use it explicitly here. If Dynaconf upstream wants to change the default alias, our object would change without our knowledge.